README.md

```diff
@@ -30,22 +30,23 @@ Curious to know how big companies are operating their kafka fleet in production?
 15. [GoTo](#goto)
 16. [Grab](#grab)
 17. [Honeycomb](#honeycomb)
-18. [LinkedIn](#linkedin)
-19. [Lyft](#lyft)
-20. [Morgan Stanley](#morgan-stanley)
-21. [Netflix](#netflix)
-22. [Pinterest](#pinterest)
-23. [Riskified](#riskified)
-24. [Robinhood](#robinhood)
-25. [Shopify](#shopify)
-26. [Slack](#slack)
-27. [Stripe](#stripe)
-28. [Uber](#uber)
-29. [Wise](#wise)
-30. [Wix](#wix)
-31. [Yelp](#yelp)
-32. [Zalando](#zalando)
-33. [Zopa Bank](#zopa-bank)
+18. [Hubspot](#hubspot)
```

Comment:
[unrelated] propose switching all this to `1.`, so new PR generates fewer changes. Markdown will automatically create the sequence anyway

Reply:
@jychen7 Sure. I think it makes sense. Could you help to change it in your PR as well?

Reply:
sure, 156b9fe23c74e6dee5c80b859c666aa298d1862c